.github/workflows/mkdocs-deploy.yml

```diff
@@ -0,0 +1,32 @@
+# GCLINE@ 23NOV2020
+name: deploy to mkdocs
+
+# the two different ways to trigger the workflow
+on:
+  # enable manual trigger button in github actions web
+  workflow_dispatch:
+
+  # enable auto trigger based on both path and branch
+  push:
+
+    # only main branch
+    branches:
+      - main
+
+    # at least one changed file must match this path
+    paths:
+      - 'docs/**'
+
+jobs:
+  build:
+    name: Deploy docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v2
+
+      - name: Deploy docs
```

Comment:
we are using [mike](https://github.com/jimporter/mike) to deploy our multi-version docs.
I think we need to have our own actions or update `mhausenblas/mkdocs-deploy-gh-pages` to support mike.

Reply:
I'll take a look at how difficult that would be ... hopefully we can still do CD